daffodil-cli/src/it/scala/org/apache/daffodil/performance/TestCLIPerformance.scala

```diff
@@ -50,7 +50,7 @@ class TestCLIPerformance {
       if (exitCode != 0)
         fail("Tests failed. Exit code: " + exitCode)
 
-      Util.expectExitCode(ExitCode.PerformanceTestError, shell)
+      Util.expectExitCode(ExitCode.Success, shell)
```

Comment:
Looks like this test is already checking the exit code. I'd suggest you delete the previous stuff and only use the new expectExitCode stuff. Other tests do something similar-we should replace that with expectExitCode in all the integration tests that do that.